packages/flutter_tools/static/index.html

```diff
@@ -1,23 +1,10 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>test Browser Host</title>
+  <title>Flutter Test Browser Host</title>
 </head>
 <body>
-  <svg id="dart" version="1.1" x="0px" y="0px" width="400px" height="400px" viewBox="0 0 400 400">
```

Comment:
These change the overall layout of tests, probably want to find the code that generates the hosted iframe so that we can force it to start at (0, 0), or else record what the offset is